src/Element/CprElement.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
```

Comment:
We have too many blank lines here. We should set up some coding standards checks (cf. https://www.drupal.org/docs/contributed-modules/code-review-module/installing-coder-sniffer#s-composer-installer-plugin).

Reply:
Fixed in https://github.com/itk-dev/os2forms_cpr_lookup/pull/6